.travis.yml

```diff
@@ -6,28 +6,23 @@ php:
 
 sudo: false
 
-env: SYMFONY_VERSION=3.2.*
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
 matrix:
   include:
     - php: 7.1
-      env: DEPS=dev SYMFONY_VERSION=3.3.*
+      env: STABILITY=dev
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.* COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.1
-      env: SYMFONY_VERSION=3.1.*
+      env: COMPOSER_FLAGS="--prefer-lowest"
   fast_finish: true
 
 before_install:
   - phpenv config-rm xdebug.ini || true
   - composer self-update
-  - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi
-  
+  - ${STABILITY:+ composer config 'minimum-stability' "$STABILITY"}
```

Comment:
So this is `if STABILITY is defined, then (set the stability in composer json)` ?